documentation/asciidoc/computers/linux_kernel/building.adoc

```diff
@@ -1,342 +1,462 @@
 
 To adjust `LOCALVERSION`, change the following line in `.config`:
 
+[source,ini]
 ----
 CONFIG_LOCALVERSION="-v7l-MY_CUSTOM_KERNEL"
 ----
 
-You can also change this setting graphically as shown in xref:linux_kernel.adoc#configuring-the-kernel[the kernel configuration instructions]. It is located in "General setup" \=> "Local version - append to kernel release".
+You can also change this setting graphically with `menuconfig` at *General setup* > *Local version - append to kernel release*. For more information about `menuconfig`, see xref:linux_kernel.adoc#configure-the-kernel[the kernel configuration instructions].
+
+==== Build and install the kernel natively
+
+Next, build and install the kernel, modules, and Device Tree blobs into the boot partition of your Raspberry Pi. This step can take a long time depending on the Raspberry Pi model you are using.
 
-==== Building the kernel
+TIP: If you don't want to install the freshly-compiled kernel onto the Raspberry Pi where you run this command, copy the compiled kernel to the boot partition of a separate boot media instead of `/boot/firmware/`.
 
-Build and install the kernel, modules, and Device Tree blobs. This step can take a long time depending on the Raspberry Pi model you are using.  For the 32-bit kernel:
+To build and install the 64-bit kernel, run the following commands:
 
-[,bash]
+[source,console]
 ----
-make -j4 zImage modules dtbs
-sudo make modules_install
-# Choose one of the following based on the kernel version
-  # For kernels up to 6.4:
-  sudo cp arch/arm/boot/dts/*.dtb /boot/firmware/
-  # For kernel 6.5 and above:
-  sudo cp arch/arm/boot/dts/broadcom/*.dtb /boot/firmware/
-sudo cp arch/arm/boot/dts/overlays/*.dtb* /boot/firmware/overlays/
-sudo cp arch/arm/boot/dts/overlays/README /boot/firmware/overlays/
-sudo cp arch/arm/boot/zImage /boot/firmware/$KERNEL.img
+$ make -j4 Image.gz modules dtbs
+$ sudo make modules_install
+$ sudo cp arch/arm64/boot/dts/broadcom/*.dtb /boot/firmware/
+$ sudo cp arch/arm64/boot/dts/overlays/*.dtb* /boot/firmware/overlays/
+$ sudo cp arch/arm64/boot/dts/overlays/README /boot/firmware/overlays/
+$ sudo cp arch/arm64/boot/Image.gz /boot/firmware/$KERNEL.img
 ----
 
-For the 64-bit kernel:
+To build the 32-bit kernel:
 
-[,bash]
+. Run the following commands to build the modules and Device tree blobs:
++
+[source,console]
 ----
-make -j4 Image.gz modules dtbs
-sudo make modules_install
-sudo cp arch/arm64/boot/dts/broadcom/*.dtb /boot/firmware/
-sudo cp arch/arm64/boot/dts/overlays/*.dtb* /boot/firmware/overlays/
-sudo cp arch/arm64/boot/dts/overlays/README /boot/firmware/overlays/
-sudo cp arch/arm64/boot/Image.gz /boot/firmware/$KERNEL.img
+$ make -j4 zImage modules dtbs
+$ sudo make modules_install
 ----
 
-NOTE: On Raspberry Pi 2, 3, 4 and 5, the `-j4` flag splits the work between all four cores, speeding up compilation significantly.
+. Depending on your kernel version, run the following command:
```

Comment:
I checked, and according to https://downloads.raspberrypi.com/raspios_armhf/release_notes.txt Bookworm was initially using kernel 6.1 but then switch to kernel 6.6 in March 2024.

Reply:
Added a link to identify your kernel version here instead!